### PR TITLE
Add tools page/dialog for running common commands for wallet repair/m…

### PR DESCRIPTION
**Pull request by mrmetech**
This is for issue #674 

image

This was originally from Dash not sure when it was added though, just pretty sure that is the root of origin.

As state in the image, this code is basically for issuing command-line startup commands from the GUI.

### CODE REVIEW

**Comment by mxaddict** [comment]
Looks nice!

File: src/qt/navcoin.cpp [hunk 1]
```diff
@@ -29,6 +29,7 @@
 #endif
 
 #include <init.h>
+#include "net.h"
```
Comment (mxaddict) [inline src/qt/navcoin.cpp]: I think this should use the <> syntax instead

**Comment by mxaddict** [comment]
Other then the comment I made above, everything else in the code looks good to me

**Comment by mrmetech** [comment]
@mxaddict ok just fixed that, thanks for letting me know

**Comment by navbuilder** [comment]
A new build of 287aa188e383914bae0cf2683f7b9d27cc626409 has completed succesfully!
Binaries available at https://build.nav.community/binaries/NAV-674

**Comment by navbuilder** [comment]
A new build of 309c36a21b629d7aacc45dc8818ca4d1836144aa has completed succesfully!
Binaries available at https://build.nav.community/binaries/NAV-674

**Pull request merged**